impress-org/givewp
# Fix: Update FFM migration to support Donation Amount

## Description



This PR updates the FFM migration to support conditional logic based on the Donation Amount block.

This also resolves an issue with the operator encoded as an HTML entity.

## Visuals



![Peek 2023-10-12 15-38](https://github.com/impress-org/givewp/assets/10858303/2be434d1-76b4-444e-a9e1-ba04e24858aa)

## Testing Instructions



- Add a custom field to a v2 form
- Set a condition for 'Donation Amount is greater than 30'
- Migrate the form

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205710844260106
